docs/src/examples/components/Tree/Performance/TreeWith60ListItems.perf.tsx

```diff
@@ -0,0 +1,70 @@
+import { Tree, Image, List } from '@fluentui/react'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+const avatars = [
+  'public/images/avatar/small/matt.jpg',
```

Comment:
Do not do any additional network request. 
Do this: https://github.com/microsoft/fluent-ui-react/blob/880790637ed2113fbb500b28f8c1126be5347061/docs/src/examples/components/Chat/Performance/ChatWithPopover.perf.tsx#L6-L14